docs/enhancements/clusterpool-inventory.md

```diff
@@ -119,26 +116,32 @@ For the VSphere case, this allows the administrator to:
   - op: replace
     path: metadata/name
     value: foo
+  - op: replace
+    path: /platform/vsphere/apiVIP
+    value: 10.0.0.10
+  - op: replace
+    path: /platform/vsphere/ingressVIP
+    value: 10.0.0.11
+
   ```
-- Add the name of ClusterDeploymentCustomization CR to `clusterPool.spec.inventory.ClusterDeploymentCustomizations` list. For ClusterDeploymentCustomization with a name `foo-cluster-deployment-customization` the clusterpool should be configured as follows
+- Add the name of ClusterDeploymentCustomization CR to `clusterPool.spec.inventory` list. For ClusterDeploymentCustomization with a name `foo-cluster-deployment-customization` the clusterpool should be configured as follows
     ```yaml
     spec:
       inventory:
-        ClusterDeploymentCustomizations:
-          - name: foo-cluster-deployment-customization
+        - name: foo-cluster-deployment-customization
     ```
   
 ### Validation
 Webhook validation will ensure that for a clusterpool
-- if `spec.inventory.ClusterDeploymentCustomizations` is specified, it is not an empty list.
+- if `spec.inventory` is specified, it is not an empty list.
 
 Other validations include
-- If `clusterpool.spec.size` is greater than length of the inventory, we put status condition on the clusterpool saying size requirements cannot be satisfied. 
+- If `clusterpool.spec.size` is greater than length of the inventory, we change `InventoryValid` change to False on the clusterpool saying size requirements cannot be satisfied. 
```

Comment:
```suggestion
- If `clusterpool.spec.size` is greater than length of the inventory, we change `InventoryValid` condition to False on the clusterpool saying size requirements cannot be satisfied. 
```

but we don’t prevent the pool from consuming the inventory it has, right? So the status.size will never reach the spec.size, but the pool will still function?

Reply:
> but we don’t prevent the pool from consuming the inventory it has, right?

correct

> So the status.size will never reach the spec.size, but the pool will still function?

correct